.github/workflows/test-end-to-end.yml

```
@@ -89,15 +95,15 @@ jobs:
       - name: Start backend
         uses: ./.github/actions/app-start
         with:
-            name: backend
-            port: 8080
-            wait: http://localhost:8080/api-docs/v1
+          name: backend
+          port: 8080
+          wait: http://localhost:8080/api-docs/v1
 
       - name: Start app
         uses: ./.github/actions/app-start
```

actionlint (1.7.12)

[error] missing input "wait" which is required by action "app-start" defined at "./.github/actions/app-start". all required inputs are "name", "port", "wait"

(action)